ssr/src/component/buttons.rs

```diff
@@ -0,0 +1,130 @@
+use leptos::*;
+
+#[component]
+pub fn HighlightedButton(
+    children: Children,
+    on_click: impl Fn() + 'static,
+    #[prop(optional)] classes: String,
+    #[prop(optional)] alt_style: Signal<bool>,
+    #[prop(optional)] disabled: Signal<bool>,
+) -> impl IntoView {
+    let on_click = move |_| on_click();
+    view! {
+        <button
+            on:click=on_click
+            disabled=disabled.get()
+            class=format!(
+                "w-full px-5 py-3 rounded-lg disabled:text-white/50 flex items-center transition-all justify-center gap-8 font-kumbh font-bold {} {}",
+                if alt_style.get() {
+                    "text-primary-600"
+                } else {
+                    "text-white"
+                },
+                classes,
+            )
+            style=format!(
+                "background: linear-gradient(73deg, {} );",
+                if disabled.get() {
+                    "#DE98BE 0%, #E761A9 33%, #7B5369 100%"
+                } else if alt_style.get() {
+                    "#FFF 0%, #FFF 1000%"
+                } else {
+                    "#DA539C 0%, #E2017B 33%, #5F0938 100%"
+                },
+            )
```

Comment:
These are not reactive

Reply:
Do they have to be reactive? beacause they dont change once rendered...

Reply:
You shouldn't use Signal if you don't want reactive behavior :P